src/labels/CustomLabels.labels

```diff
@@ -847,6 +847,22 @@
         <shortDescription>Button label for multiple EDA Settings tabs</shortDescription>
         <value>Edit</value>
     </labels>
+    <labels>
+        <fullName>stgBtnRefreshAdminNames</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button label for refreshing Administrative Account Names</shortDescription>
```

Comment:
Not sure if we want get Doc involved on these description. Let me know if you already have.

Reply:
I tried to keep in sync with what we'd been doing historically. I think the risk is low given that this only appears in setup with custom labels and not really 'in the customer's face'
I have not involved doc as I felt that would be overkill.